modules/kafka/kafka.go

```diff
@@ -46,14 +63,34 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 
 // Run creates an instance of the Kafka container type
 func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*KafkaContainer, error) {
+	listeners := []string{
+		fmt.Sprintf("LOCALHOST://0.0.0.0:%d", PublicLocalhostPort.Int()),
+		fmt.Sprintf("HOST_DOCKER_INTERNAL://0.0.0.0:%d", PublicDockerHostPort.Int()),
+		fmt.Sprintf("CONTAINER_NAME://0.0.0.0:%d", NetworkInternalContainerNamePort),
+		fmt.Sprintf("CONTAINER_ID://0.0.0.0:%d", NetworkInternalContainerIdPort),
+		fmt.Sprintf("BROKER://0.0.0.0:%d", BrokerToBrokerPort),
+		fmt.Sprintf("CONTROLLER://0.0.0.0:%d", ControllerPort),
```

Comment:
suggestion: looks like these could all be simplified to strings concatenation with a type change.

Reply:
I think with string concat the ease of reading and verifying the LISTENER:// <> Port pairings will be maintained; I'll see what that looks like.

Reply:
yep that's where I was going:
```go
	listeners := strings.Join([]string{
		"BROKER://0.0.0.0:"+brokerToBrokerPort,
		"CONTROLLER://0.0.0.0:"+controllerPort,
...
	})
```